components/cdc/src/endpoint.rs

```diff
@@ -1203,8 +1204,8 @@ impl<T: 'static + RaftStoreRouter<RocksEngine>> Runnable for Endpoint<T> {
                 cb,
             } => {
                 debug!("downstream was initialized"; "downstream_id" => ?downstream_id);
-                downstream_state
-                    .compare_and_swap(DownstreamState::Uninitialized, DownstreamState::Normal);
+                let _ = downstream_state
```

Comment:
Is it safe to ignore error? @5kbpers @overvenus

Reply:
yes

Reply:
If so, why not just use `store`?

Reply:
Could you unwrap it?

Reply:
Because it could be `Stopped` state

Reply:
Is it safe to transit state from `Stopped` to `Normal`?

Reply:
No, so we should not unwrap it here